setup/create-wp-project/src/basics/arguments.js

```diff
@@ -0,0 +1,74 @@
+const chalk = require('chalk'); // eslint-disable-line import/no-extraneous-dependencies
+const inquirer = require('inquirer');
+const { log, label } = require('./misc');
+
+/**
+ * Output a summary for all user-provided answers and ask for a confirmation.
+ *
+ * @param {array} answers Array of user-provided answers.
+ */
+const summary = async(answers) => {
+  log('');
+  log(label('Summary: '));
+  Object.keys(answers).forEach((key) => {
+    log(`- ${key}: ${chalk.cyan(answers[key])}`);
+  });
+
+  const { confirmSummary } = await inquirer.prompt({
+    name: 'confirmSummary',
+    type: 'confirm',
+    message: 'Looks good?',
+  });
+
+  return confirmSummary;
+};
+
+/**
+ * Should prompt the user for all scriptArguments.
+ *
+ * TODO: Only prompt for things not provided as arguments on command line.
```

Comment:
If this `TODO` is not implemented, remove it from here and open an issue. 

Inline `TODO`s are never done 😄

Reply:
> If this `TODO` is not implemented, remove it from here and open an issue.
> 
> Inline `TODO`s are never done 😄

That's what I just implemented in this branch, forgot to remove the TODO tho, good catch, thx!